docs/shared/distributed-cron.md

```diff
@@ -0,0 +1,42 @@
+You can turn any Temporal Workflow into a repeatedly initiated Workflow. Two choices:
+
+- start a Workflow using the Temporal CLI with an optional cron schedule using the `--cron` argument
+- (recommended) Supply a cron schedule when starting the Workflow using the CronSchedule
```

Comment:
```suggestion
2. **Recommended**: Supply a cron schedule when starting the Workflow Execution using the CronSchedule.
```

I know you like to front load recommendations, which I support. I just want to find a way for us to do it that looks a bit cleaner such that we can build style guidance around it.
I don't want to block on this one, as I need to think about how to achieve this long term.